tests/integration/lib/helpers.rb

```diff
@@ -24,7 +24,7 @@ def start_mailhog
       `MailHog --version`
       spawn "MailHog"
     rescue Errno::ENOENT
-      puts "MailHog is not installed (or not in the PATH)".yellow
+      # Ignored
```

Comment:
Why is the mailhog error ignored ? The comment could be better IMHO :D

Reply:
We are launching MailHog as a docker service on github actions (https://github.com/cozy/cozy-stack/pull/2123/files#diff-7327122c763912b429300af1b6158a12R17-R21), but you are right, the comment should be better